receiver/prometheusremotewritereceiver/receiver.go

```diff
@@ -263,8 +263,10 @@ func (prw *prometheusRemoteWriteReceiver) translateV2(_ context.Context, req *wr
 			hashedLabels := xxhash.Sum64String(ls.Get("job") + string([]byte{'\xff'}) + ls.Get("instance"))
 
 			if existingRM, ok := prw.rmCache.Get(hashedLabels); ok {
+				fmt.Println("capturing from existing rm")
```

Comment:
Generally we don't do `fmt.Prinln()` but use the existing logging library,
Also not sure these log line provide much value in the first place outside of local testing.